services/consensusalgo/benchmarkconsensus/test/harness.go

```diff
@@ -89,7 +75,7 @@ func newHarness(tb testing.TB, isLeader bool) *harness {
 		blockStorage:     blockStorage,
 		consensusContext: consensusContext,
 		reporting:        log,
-		config:           cfg.OverrideNodeSpecificValues(":8080", 0, nodeKeyPair.NodeAddress(), nodeKeyPair.PrivateKey(), ""),
```

Comment:
this leaves only one other instance where `OverrideNodeSpecificValues()` is called. 
It follows the only invocation of `config.ForE2E()` (in `e2e/network.go:85`).

Consider joining `OverrideNodeSpecificValues()` and  `ForE2E()`  into a single function (`ForE2E()`) and removing `OverrideNodeSpecificValues()` from the interface `OverridableConfig`

Reply:
subsequent PR